newrelic-infrastructure-bundle.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: newrelic-infrastructure-bundle
+  version: 3.2.2
+  epoch: 0
+  description: New Relic Infrastructure containerised agent bundle
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-baselayout
+      - busybox
+      - build-base
+      - go
+      - ca-certificates-bundle
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/newrelic/infrastructure-bundle
+      tag: v${{package.version}}
+      expected-commit: ece3dc47380c31d131a17d2b3b9f42f9811545fc
+      destination: ${{package.name}}
+
+  - working-directory: ${{package.name}}
+    pipeline:
+      - runs: |
+          go run downloader.go
+
+          case "${{build.arch}}" in
```

Comment:
Can this be based on `go env GOARCH`? Then when we add a new arch we don't have to remember to add a case here.

Reply:
Unfortunately, that `downloader.go` [does not allow to pass](https://github.com/newrelic/infrastructure-bundle/blob/ece3dc47380c31d131a17d2b3b9f42f9811545fc/downloader.go#L57-L63) custom arch and not respect the `GOARCH`. Force downloads the all archs: `amd64`, `arm64` and `arm`.